src/EditorFeatures/Core/RenameTracking/RenameTrackingOptions.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.CodeAnalysis.Options;
+
+namespace Microsoft.CodeAnalysis.RenameTracking
+{
+    internal static class RenameTrackingOptions
+    {
+        public static readonly PerLanguageOption2<bool> RenameTrackingPreview = new("dotnet_rename_tracking_preview", defaultValue: true);
```

Comment:
>dotnet_rename_tracking_preview [](http://example.com/codeflow?start=85&length=30)

Not clear from the option name what it does. Does it mean "show preview for rename tracking", or is it a "preview" feature?

Reply:
<img width="687" alt="image" src="https://user-images.githubusercontent.com/24360909/220491702-db4c43c1-adf2-40fd-b6d0-c985bdcca73b.png">
Will change the name here